src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +85,13 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    public void addTag(Tag tag){
```

Comment:
You can make use of an enum to keep track of the two values add/delete or even a boolean, instead of a string.